<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/downloadcontrolbarwidget.ui -->
<!-- Phiên bản: 1.0 (Mới) -->
<ui version="4.0">
 <class>DownloadControlBarWidget</class>
 <widget class="QWidget" name="DownloadControlBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>+ Thêm từ Clipboard</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Bắt đầu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>Dừng</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
